layouts/partials/head.html

```diff
@@ -6,6 +6,7 @@
 {{- else -}}
 <meta name="ROBOTS" content="NOINDEX, NOFOLLOW">
 {{- end -}}
+{{- if ne hugo.Environment "local" -}}
```

Comment:
In the `Makefile`, we use the environment name _development_. Is that appropriate here?

I'm not sure what _local_ would signify that's different from _development_?

I reckon that we should turn off Google Analytics for any environment except:
- _nonprod_
- _production_

How does that sound?

Reply:
That sounds great. I would have done exactly that but as I said I wasn't sure if it was appropriate for development/preview.

Or I mean, I was totally sure it was objectively correct, but I wanted to make the change with the most minimal possible impact on anything rather than telling you guys which lifecycle stages should have gtags.